eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -158,16 +160,31 @@ public void exec() throws IOException {
                 this.targetDir.toPath().resolve(TranspileMojo.DIR),
                 TranspileMojo.EXT
             );
+            final String src = tojo.get(AssembleMojo.ATTR_EO);
             if (
                 target.toFile().exists()
                     && target.toFile().lastModified() >= file.toFile().lastModified()
+                    && target.toFile().lastModified() >= Paths.get(src).toFile().lastModified()
             ) {
                 Logger.info(
                     this, "XMIR %s (%s) were already transpiled to %s",
                     Save.rel(file), name, Save.rel(target)
                 );
             } else {
-                saved += this.transpile(input, target);
+                final List<Path> paths = this.transpile(src, input, target);
+                for (final Path path : paths) {
+                    this.transpiledTojos.value()
+                        .add(
+                            String.format(
```

Comment:
@Karina5005 this looks wrong. We are not supposed to save multi-live values into Tojos file (since it's essentially a CSV). Why do we need this strange value construct here?

Reply:
I wanted to use eo as an id, but it's not possible because there are many java files for one eo, so I decided to use pair `eo, java` as unique key. Now I fixed it and use java as id, will it be better?